fluent-bit.yaml

```diff
@@ -0,0 +1,64 @@
+package:
+  name: fluent-bit
+  version: 2.0.8
+  epoch: 0
+  description: Fast and Lightweight Log processor and forwarder
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: Apache-2.0
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - bison
+      - build-base
+      - busybox
```

Comment:
What's this for? A shell to build with?

Reply:
Those are build dependency. I havent declared any runtime dependency yet